### DISCUSSION TITLE
Universal covering spaces, semi-locally simply connected, etc...

### DISCUSSION THREAD

**Discussion by GeoffreySangston** [pi-base/data#817] [Ideas]
(I apologize if this has already been discussed. I looked around and couldn't seem to see anything about it. Also, I've been using pi-base pretty consistently since college and think it's an incredibly helpful tool, so I just want to say thanks!)

I don't see the following properties. Is there a good reason for omitting these?

- [Contractible](https://en.wikipedia.org/wiki/Contractible_space)
- Locally contractible (on the same page as contractible)
- Strongly locally contractible (on the same page as contractible)
- [Simply connected](https://en.wikipedia.org/wiki/Simply_connected_space#Definition_and_equivalent_formulations)
- [Semi-locally simply connected](https://en.wikipedia.org/wiki/Semi-locally_simply_connected)
- Existence of a universal cover (Corollary 82.2 of Munkres says this is equivalent to path connected + locally path-connected + semi-locally simply connected)
- [Locally simply connected](https://en.wikipedia.org/wiki/Locally_simply_connected_space)
- [Acyclic](https://en.wikipedia.org/wiki/Acyclic_space)

(Edit: I think it's safe to say the standard definitions for the following depend on having a basepoint, since they depend on statements about the higher homotopy groups. So maybe that is a justification for not having them.)
- [Weakly contractible](https://en.wikipedia.org/wiki/Weakly_contractible)
- [Being an Eilenberg-MacLane space](https://en.wikipedia.org/wiki/Eilenberg%E2%80%93MacLane_space)
- [Aspherical](https://en.wikipedia.org/wiki/Aspherical_space)

And one could have properties of the fundamental group, in analogy to simply connected. For example, Hatcher has the following on page 49, "It is a theorem of [Shelah 1988] that for a path-connected, locally path-connected compact metric space X , π1(X) is either finitely generated or uncountable."

If these were added, pi-base should also have the [Hawaiian earring space](https://en.wikipedia.org/wiki/Hawaiian_earring). (There does appear to be a controversy around this name. Hatcher uses the descriptive name _The Shrinking Wedge of Circles_.)  And perhaps there should be the [rose with infinitely many petals](https://en.wikipedia.org/wiki/Rose_(topology)), which comes up when discussing the previous space. And I assume there's a whole canon of counterexamples here. I'd like to see spaces like the [pseudocircle](https://en.wikipedia.org/wiki/Pseudocircle), but perhaps that requires being able to make a more complex kind of query than pi-base desires to support (e.g., specifying the homotopy groups in some complex way).

**Comment by StevenClontz** [comment]
I'll reply in more detail later, but we recently added the earring space: https://topology.pi-base.org/spaces/S000201 (and had similar conversations about the name). And we have https://topology.pi-base.org/spaces/S139 as well for comparison.

**Reply by StevenClontz** [reply]
But in general, any property preserved by homeomorphism that's of interest to researchers in general or set-theoretic topology is a good fit here.

**Reply by GeoffreySangston** [reply]
Oh I see. I suppose I need to clear my cache because https://topology.pi-base.org/spaces/S000201 gives
404
Property not found S000201
on Chrome. I opened it in Safari and it does show up.

So now I'm not sure which of those properties are already supported. I'll double check later.

**Reply by StevenClontz** [reply]
Use sync or reset at https://topology.pi-base.org/dev to get updates (we need to make it more visible).

**Comment by prabau** [comment]
@StevenClontz Can this "Discussion" be moved to an "issue" maybe?

**Comment by StevenClontz** [comment]
Now an issue at https://github.com/pi-base/data/issues/818
